vllm/attention/backends/xformers.py

```diff
@@ -50,34 +155,34 @@ def __init__(
 
         assert self.num_heads % self.num_kv_heads == 0
         self.num_queries_per_kv = self.num_heads // self.num_kv_heads
-        suppored_head_sizes = PagedAttentionImpl.get_supported_head_sizes()
+
+        suppored_head_sizes = PagedAttention.get_supported_head_sizes()
         if head_size not in suppored_head_sizes:
             raise ValueError(
                 f"Head size {head_size} is not supported by PagedAttention. "
                 f"Supported head sizes are: {suppored_head_sizes}.")
 
+        # Some old AMD GPUs do not support xFormers nor FlashAttention.
+        # As a temporary workaround, we use naive PyTorch implementation
+        # of attention.
         self.use_ref_attention = _check_use_ref_attention()
```

Comment:
Let's separate this out to another attention backend?

Reply:
While I don't like this part of code, I feel there will be too many duplicates between the backends if we create another one for this. WDYT?

BTW, I fixed the comment since it turns out that this is for new consumer-grade AMD AI GPUs, not for old GPUs. Also, changed `ref_attention` to `naive_attention`.